app/views/admin/users/index.html.haml

```diff
@@ -0,0 +1,23 @@
+.row
+  .col-lg-12
+    = breadcrumbs [['Admin', admin_path],
+                   ['Users']]
+
+    = title 'Users'
+
+    .pull-right
+      %ul.nav.nav-pills
```

Comment:
Nope, there isn't; and we're only using nav pills on one other template, so I'm not sure it makes sense to add one for now...